### DISCUSSION TITLE
Filter pseudo-releases

### DISCUSSION THREAD

**Discussion by emiham** [beetbox/beets#5004] [Q&A]
Is there a way in which I can filter pseudo-releases from my search results? Sometimes they look identical in beets, meaning I have to open MusicBrainz to figure out which one is right. 

I did some research and found this comment: https://github.com/beetbox/beets/issues/654#issuecomment-841909730, but it doesn't seem like the field I would want is available. Is there another way to go about this?

**Comment by RollingStar** [comment]
You probably already found this, but looks like pseudo release is a status, and status isn't in the distance_weights, so it can't(?) be used in distance_weights. Or can it? Try adding a `status: 99` key to distance_weights. The code looks like it might take _any_ key you specify and use it.

But distance weights isn't the way to do this, that would just de-rate statuses that differ if the status was already set on what you were importing. "preferred" would be, but the code isn't dynamic currently. It would have to be hard coded in, or rewritten.

https://musicbrainz.org/ws/2/release/1c981462-1fe0-422d-848d-0e7c6ead5ba1?inc=artist-credits+labels+discids+recordings&fmt=json

https://github.com/beetbox/beets/blob/4a6668a0cd3f988779cbedf71bcb7f5672205ff1/beets/autotag/match.py#L236

**Reply by emiham** [reply]
It took me a while before I had new stuff to tag to try this on, but it seems `status: 99` in `distance_weights` doesn't work.
